src/Compilers/CSharp/Portable/FlowAnalysis/EmptyStructTypeCache.cs

```diff
@@ -215,12 +215,17 @@ private FieldSymbol GetActualField(Symbol member, NamedTypeSymbol type)
                     return (!eventSymbol.HasAssociatedField || ShouldIgnoreStructField(eventSymbol, eventSymbol.Type)) ? null : eventSymbol.AssociatedField.AsMember(type);
                 case SymbolKind.Property:
                     // PROTOTYPE(semi-auto-props): Review other event associated field callers and see if we have to do anything special for properties.
+
                     // Backing field for semi auto props are not included in GetMembers.
-                    // PROTOTYPE(semi-auto-props): This condition is always false after modifiying BackingField implementation. Figure out a
-                    // test that needs this.
-                    if (member is SourcePropertySymbol { BackingField: SynthesizedBackingFieldSymbol { IsCreatedForFieldKeyword: true } backingField })
+                    if (member is SourcePropertySymbol property)
                     {
-                        return backingField;
+                        // PROTOTYPE(semi-auto-props): property.BackingField be 'IsCreatedForFieldKeyword' only when it has initializer.
+                        // Add a test for that when initializers are supported.
+                        var backingField = property.BackingField ?? property.FieldKeywordBackingField;
```

Comment:
>var backingField = property.BackingField ?? property.FieldKeywordBackingField; [](http://example.com/codeflow?start=24&length=78)

A couple of questions for this change:
1. Is the behavior change observable in existing tests?
2. Can't we simply do ```property.FieldKeywordBackingField is not null``` below?

Reply:
> Is the behavior change observable in existing tests?

Yes. For example, `TestPropertyNotAssignedInStructConstructor`. There is no definite assignment error without this change.

---

> Can't we simply do property.FieldKeywordBackingField is not null and var backingField below?

Checking `FieldKeywordBackingField` alone should work. Thanks.